Source/WebKit/UIProcess/ios/WKContentView.mm

```diff
@@ -156,7 +183,8 @@ @implementation WKContentView {
 
     Lock _pendingBackgroundPrintFormattersLock;
     RetainPtr<NSMutableSet> _pendingBackgroundPrintFormatters;
-    IPC::Connection::AsyncReplyID _pdfPrintCallbackID;
+    IPC::Connection::AsyncReplyID _printRenderingCallbackID;
+    _WKPrintRenderingState _printRenderingState;
```

Comment:
Since this variable simply describes the nature of the callback ID above, I don't think we need `_WKPrintRenderingStateNone`. A zero `_printRenderingCallbackID` would imply that.

I would also rename `_printRenderingState` to reflect that it describes the callback, since it doesn't appear as though it's consistently updated everywhere (particularly in the background thread cases).

Reply:
@pxlcoder any suggestions as to what to rename `_printRenderingState` to? `_printRenderingCallbackState`?

Reply:
`_printRenderingCallbackType`?